ui/widgets/sidebar/src/components/sidebar-new.tsx

```diff
@@ -182,13 +182,7 @@ const Sidebar: React.FC<ISidebarProps> = props => {
           {footerIcons.map((icon, idx) => (
             <div key={icon.name + idx} className={tw('mr-4')}>
               <a href={icon.link} target="_blank" rel="noreferrer noopener">
-                <div
-                  className={tw(
-                    'w-10 h-10 flex items-center justify-center rounded-full bg-grey9 dark:bg-grey3',
-                  )}
-                >
-                  <Icon type={icon.name} isCustomIcon={true} plain={true} accentColor={true} />
-                </div>
+                <AppIcon placeholderIconType={icon.name} size="sm" plain accentColor />
```

Comment:
Can we have both plain and accentColor props set in one icon?

Reply:
I am not seeing the need to have this plain prop. The default can be set as plain. That will be the base style. Then I will convert the svg's for all custom icons to the once we have on the design system that will make the Icon component leaner and cleaner. All this will be coming in my next PR.